### PR TITLE
fix: we are Artsy, our beta app is called `Δrtsy`, and our appstore app is called `Artsy` again

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves []

### QA Test Case(s)

| Test Case | Feature | Environment | Acceptance Criteria | Setup Instructions/Link |
| --------- | :-----: | ----------: | ------------------: | ----------------------: |
|           |         |             |                     |                         |

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: ios/Artsy/App_Resources/Info.plist [hunk 1]
```diff
@@ -29,6 +27,8 @@
 	<string>${EXECUTABLE_NAME}</string>
 	<key>CFBundleIdentifier</key>
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleDisplayName</key>
+	<string>$(DISPLAY_NAME)</string>
```
Comment (gkartalis) [inline ios/Artsy/App_Resources/Info.plist:31]: 👏👏👏

File: .secrets.baseline [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (gkartalis) [inline .secrets.baseline:2]: 👀 was this on purpose?

**Review by gkartalis — APPROVED** [review]

File: .secrets.baseline (hunk 2 above)
Comment (pvinis) [reply .secrets.baseline:2]: yea.. the ci is on 1.2.0 lol. and so this PR is not getting green because the base commit is from a time before 1.2.0 :p

Comment (gkartalis) [reply .secrets.baseline:2]: noice! Feel free to merge/release whenever

**Pull request merged**